src/playlist-maker.js

```diff
@@ -78,6 +78,38 @@ var playlistMaker = function(player, plist) {
     return currentIndex;
   };
 
+  // item can be either
+  //  * a string
+  //  * an array of sources, which are either strings or {src, type} objects
+  //  * a playlist item
+  playlist.contains = function contains(item) {
```

Comment:
Just realized that I can't just convert this `contains` to `indexOf` because it uses `isInSources` helper which just returns a boolean. Might have to rewrite both. Or just add an `indexInSources` or something.